modules/openy_features/openy_node/modules/openy_node_event/templates/event-contact-info.html.twig

```diff
@@ -0,0 +1,9 @@
+<div class="info-col contact col-md-4">
```

Comment:
Add docblock with list of all variables coming to the template. See node.html.twig